lib/longrunning.js

```diff
@@ -294,7 +294,8 @@ Operation.prototype.startPolling_ = function() {
       }
 
       if (!result) {
-        if (rawResponse.metadata.value !== previousMetadataBytes) {
+        if (Buffer.compare(
+              rawResponse.metadata.value, previousMetadataBytes) !== 0) {
```

Comment:
just curious but why not `rawResponse.metadata.value.compare(previousMetadataBytes)` ?